app/supplier_utils.py

```diff
@@ -94,7 +94,6 @@ def update_open_declarations_with_company_details(db, supplier_id, updater_json,
                 supplier.companies_house_number or supplier.other_company_registration_number
             ),
             supplier_constants.KEY_DUNS_NUMBER: supplier.duns_number,
-            supplier_constants.KEY_VAT_NUMBER: supplier.vat_number,
```

Comment:
We can also remove the KEY_VAT_NUMBER line from `supplier_constants.py`.

Reply:
👍